cypress/integration/app_spec.ts

```diff
@@ -38,6 +50,35 @@ describe("Home Page", () => {
   });
 });
 
+describe("Pagination element functionality", () => {
+  beforeEach(() => {
+    setupServerMocks();
+  });
+
+  it("checks if next page button shows next page of results", () => {
+    cy.visit("/search?q=hello");
+    cy.get(`[data-e2e-id="SoundList-track-name"]`)
+      .first()
+      .should("be.visible")
+      .should("have.text", "hello");
+    cy.get(`[data-e2e-id="SoundList-track-url"]`)
+      .first()
+      .should("be.visible")
+      .should("have.attr", "href", "/sound/123");
```

Comment:
We first check to see that the first page of search results for "hello" show up properly.